product/models.py

```diff
@@ -0,0 +1,98 @@
+from django.db import models
+
+from core      import models as core_models
+
+class Product(core_models.TimeStampedModel):
+    title        = models.CharField(max_length=300)
+    thumb_image  = models.URLField(max_length=1000)
+    price        = models.IntegerField()
+    seller       = models.ForeignKey('user.Seller', on_delete=models.CASCADE)
+    delivery     = models.ForeignKey('product.Delivery', on_delete=models.CASCADE)
+    sale         = models.ForeignKey('product.Sale', on_delete=models.CASCADE, null=True)
+    category     = models.ForeignKey('product.Category', on_delete=models.CASCADE)
+    sub_category = models.ForeignKey('product.SubCategory', on_delete=models.CASCADE)
+    size         = models.ManyToManyField('product.Size', through = "ProductSize", related_name='sizes')
+    color        = models.ManyToManyField('product.Color', through = "ProductColor", related_name='colors')
+
+    class Meta:
+        db_table = 'products'
+
+    def __str__(self):
+        return self.title
+
+class ProductDetail(core_models.TimeStampedModel):
+    image_url   = models.URLField(max_length=1000)
+    description = models.CharField(max_length=500)
+    product     = models.ForeignKey('product.Product', on_delete=models.CASCADE)
+
+    class Meta:
+        db_table = 'product_details'
+
+class Sale(models.Model):
+    sale_ratio = models.DecimalField(max_digits=3, decimal_places=2)
+
+    class Meta:
+        db_table = 'sale_ratios'
+    
+    def __str__(self):
+        return self.sale_ratio
+
+class Delivery(models.Model):
+    delivery_code = models.IntegerField()
```

Comment:
송장번호를 임의로 저장해서 쓰시려는건가요?
그러면 0000으로 시작하는 송장번호는 쓸수가 없죠.
```suggestion
    delivery_code = models.CharField()
```
로 하거나 랜덤코드 생성이라면 UUIDField 필드도 있으니 참고해보세요.
https://docs.djangoproject.com/en/3.1/ref/models/fields/#uuidfield

Reply:
order 테이블의 number에 UUID 필드 적용해보고, 랜덤규칙 커스터마이징할지는 좀더 상의해보겠습니다.